aea/helpers/profiling.py

```diff
@@ -50,6 +50,11 @@ def get_current_process_memory_usage() -> float:
         d = win32process.GetProcessMemoryInfo(win32process.GetCurrentProcess())  # type: ignore
         return 1.0 * d["WorkingSetSize"] / 1024 ** 2
 
+    def get_peak_process_memory_usage() -> float:
+        """Get current process memory usage in MB."""
+        d = win32process.GetProcessMemoryInfo(win32process.GetCurrentProcess())  # type: ignore
+        return 1.0 * d["PeakWorkingSetSize"] / 1024 ** 2
```

Comment:
The magic values used here and below should be declared as constants so their meaning is explicit.